tracing/src/test/java/com/palantir/tracing/TracerTest.java

```diff
@@ -58,7 +59,7 @@ public final class TracerTest {
     @Captor
     private ArgumentCaptor<Span> spanCaptor;
 
-    @AfterEach
+    @BeforeEach
     public void before() {
```

Comment:
It is unfortunate the checkstyle rule doesn't catch this shape of mismatch. We could add an error-prone check for mismatched `@BeforeEach`/`@AfterEach` & `@BeforeClass`/`@AfterClass` annotations, but not sure how many occurrences it would catch & prevent.

https://github.com/palantir/tracing-java/blob/abc119d93909ea292bb2226908316e21d2cd5d61/.baseline/checkstyle/checkstyle.xml#L352-L353